.mocharc.cjs

```diff
@@ -1,3 +1,5 @@
 module.exports = {
-  timeout: 90000
+  timeout: 90000,
+  bail: true,
```

Comment:
I ended up rolling this one back since it was a little frustrating to have to keep re-running specs and then getting halted when one fails, re-starting the runner, bailing at the next failure, etc.  It can be nice to let things keep running to see all the failures in totality during a single run.

Perhaps this could be useful as a CI only flag?  So we don't keep wasting build time after a failure?  🤔